deepinv/datasets/set14.py

```diff
@@ -0,0 +1,137 @@
+from typing import Any, Callable
+import os
+
+from PIL import Image
+import torch
+
+from deepinv.datasets.utils import (
+    calculate_md5_for_folder,
+    download_zipfile,
+    extract_zipfile,
+)
+
+
+class Set14HR(torch.utils.data.Dataset):
+    """Dataset for `Set14 <https://paperswithcode.com/dataset/set14>`_.
+
+    The Set14 dataset is a dataset consisting of 14 images commonly used for testing performance of Image Super-Resolution models.
```

Comment:
Image Super-Resolution models -> image reconstruction algorithms

Reply:
Add a line saying "Images have sizes ranging from 276×276 to 512×768 pixels."

Reply:
Done